### ISSUE TITLE
Missing "locality" prompts in parts of the BabelEdits dataset

### ISSUE THREAD

**Issue by AnsonJIang-Pku** [umanlp/babeledits#1]
Hello, 

when I reproducing your code, I found not all the entries in the BabelEdits dataset contain the `"locality"` prompts.

For example, only 709/1042 test set entries have `"locality"` prompts, (7747/11498 for training set, 330/480 for validation set). So this might cause big problems when evaluating locality, and even affects the results in your paper.

Could you please clarify:
- whether the missing `"locality"` prompts are intentional?

- if not, whether a fixed version of the dataset will be released?

Thank you for your great work!
